datamegh/__init__.py

```diff
@@ -15,7 +15,7 @@
 }
 config: object = {
     "logging": {
-        "level": log_level_map.get(environ.get("LOG_LEVEL")) or DEBUG,
+        "level": log_level_map.get(environ.get("LOG_LEVEL", "debug")),
```

Comment:
How could this solve 😂

Reply:
@sbimochan This was a logical issue. What happens when `environ.get("LOG_LEVEL")` resolves to `None` you can't get `log_level_map.get(None)` because it expects `str` value for the key and `None` isn't expected.

Reply:
Wow pyRight is great